### DISCUSSION TITLE
ideavim sneak function When the character is at the end, can the second char be a space?

### DISCUSSION THREAD

**Discussion by ifnk** [JetBrains/ideavim#850] [General]
![image](https://github.com/JetBrains/ideavim/assets/39295458/4eb352d3-f24a-4731-a2e7-926aaf2240f2)
sneak is two characters to confirm a jump position, but some characters are at the end, and the symbols are } ] ), which are difficult to press. I think you only need to press one of the characters at the end, and use the other one. Is it possible to replace spaces with spaces?

**Comment by liango2** [comment]
press }Enter should stay on } but is the next line now
